core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/repository/DesignMetadataRepository.java

```diff
@@ -72,6 +73,9 @@ public void save(Script script) {
                 script.setSecurityGroupKey(publicSecurityGroup.getMetadataKey());
                 script.setSecurityGroupName(publicSecurityGroup.getName());
             }
+
+            script.getVersion().setCreatedBy("admin");
+            script.getVersion().setCreatedAt(LocalDateTime.now().toString());
```

Comment:
this causes the creation timestamp to be overriden when updating a script through CLI

Reply:
The update of a Script should not modify the created by/at information of a script